vm/src/dictdatatype.rs

```diff
@@ -488,6 +488,17 @@ impl<T: Clone> Dict<T> {
         }
     }
 
+    pub fn next_entry(&self, position: &AtomicCell<usize>) -> Option<(PyObjectRef, T)> {
```

Comment:
Maybe this could be named `next_entry_atomic`?

Reply:
Is it better to revert `_next_entry` to `next_entry` when I change the function name to `next_entry_atomic`?